pom.xml

```diff
@@ -817,12 +817,21 @@
       <artifactId>netty-transport</artifactId>
       <version>${netty.version}</version>
     </dependency>
+    <!-- Lets add this as optional dependency so users can include it for GSO support -->
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-native-epoll</artifactId>
+      <version>${netty.version}</version>
+      <optional>true</optional>
+    </dependency>
```

Comment:
we could also consider adding io_uring here and provide an implementation for GSO as well under the hood

Reply:
@chrisvest WDYT ^^

Reply:
Sounds good!